ocs_ci/ocs/ui/mcg_ui.py

```diff
@@ -0,0 +1,310 @@
+        self.do_click_by_id(nss_name_lst[0])
+
+        self.do_click(self.bucketclass["bs_dropdown"])
+        self.do_click_by_id(bs_name_lst[0])
+
+        self.do_send_keys(self.bucketclass["ttl_input"], "5")
+        self.do_click(self.bucketclass["ttl_time_unit_dropdown"])
+        self.do_click(self.bucketclass["ttl_minute_time_unit_button"])
+
+    set_namespacestore_policy = {
+        "single": set_single_namespacestore_policy,
+        "multi": set_multi_namespacestore_policy,
+        "cache": set_cache_namespacestore_policy,
+    }
+
+    def create_namespace_bucketclass_ui(
+        self, bc_name, policy, nss_name_lst, bs_name_lst
+    ):
+        """
+        Create a namespace BC via the UI
+
+        bc_name (str): The name to grant the OBC
+        policy (str): The policy type to use. Single/Multi/Cache
+        nss_name_lst (list[str]): A list of namespacestore names to be used by the bucketclass
+        bs_name_lst (list[str]): A list of namespacestore names to be used by the bucketclass
+
+        """
+        self.navigate_to_ocs_operator_page()
+
+        logger.info("Enter the BC section")
+        self.do_click(self.ocs_loc["bucketclass_page"])
+
+        logger.info("Create a new BC")
+        self.do_click(self.generic_locators["create_resource_button"])
+
+        logger.info("Pick type")
+        self.do_click(self.bucketclass["namespace_type"])
+
+        logger.info("Enter BC name")
+        self.do_send_keys(self.bucketclass["bucketclass_name"], bc_name)
+        self.do_click(self.generic_locators["submit_form"])
+
+        logger.info(f"Pick policy ({policy})")
+        self.do_click(self.bucketclass[f"{policy}_policy"])
+        self.do_click(self.generic_locators["submit_form"])
+
+        logger.info("Pick resources")
+        if policy == "cache":
+            self.set_namespacestore_policy[policy](self, nss_name_lst, bs_name_lst)
+        else:
+            self.set_namespacestore_policy[policy](self, nss_name_lst)
+        self.do_click(self.generic_locators["submit_form"])
+
+        logger.info("Submit")
+        self.do_click(self.generic_locators["submit_form"])
+
+    def delete_bucketclass_ui(self, bc_name):
+        """
+        Delete a BC via the UI
+
+        bc_name (str): Name of the BC to be deleted
+
+        """
+        self.navigate_to_ocs_operator_page()
+
+        logger.info("Enter the BC section")
+        self.do_click(self.ocs_loc["bucketclass_page"])
+
+        logger.info("Search for the BS")
+        self.do_send_keys(self.generic_locators["search_resource_field"], bc_name)
+
+        logger.info("Open BC kebab menu")
+        self.do_click(self.generic_locators["kebab_button"])
+
+        logger.info("Click on 'Delete Bucket Class'")
+        self.do_click(self.generic_locators["delete_resource_kebab_button"])
+
+        logger.info("Confirm BC Deletion")
+        self.do_click(self.generic_locators["confirm_action"])
+
+
+class ObcUI(PageNavigator):
```

Comment:
replace `ObcUI` with `OBCUI`

Reply:
I don't know whether that's a good idea - adhering to PEP-8 is important, but it makes the class name unreadable for people who do not know the acronyms used